specs/WinRT/WinRTAPIContracts.md

```diff
@@ -22,18 +22,22 @@ WinRT APIs.
    1. One contract per feature/component/sub-system
    2. Contracts will not span across a transport package/repo
 2. **Contract Version**<BR>
-Contracts versions are a single number with no minor version, no "x.y".  The contract version is
-frozen on a stable release.  Any changes after a stable release requires the version to be incremented.
+Contracts versions are a single number with no minor version, no "x.y". The contract version is
+frozen on a stable release. Any changes after a stable release requires the version to be incremented.
 3. **Enforcement**<BR>
 The build system has gates that enforce contract versioning to verify that contract is present,
  up to date, and there are no conflicts between APIs.
 4. **Contract Migration**<BR>
 This spec does not deal with contract migration. A separate spec will be available for contract
 migration if, and when, it is needed.
-5. **Internal contracts**<BR>
-All internal WinRT APIs have a contract, and the version is 1. The version is never updated.
-6. **Contract naming**<BR>
-Contract names must end with the word `Contract`.
+5. **Contract naming**<BR>
```

Comment:
Please no HTML (BR tag). To get a soft return in MD end the line with two spaces

Reply:
That plays poorly with editors that strip trailing whitespace